updater/osv/osv.go

```diff
@@ -626,6 +627,34 @@ func (e *ecs) NewVulnerability() *claircore.Vulnerability {
 	return &e.Vulnerability[i]
 }
 
+// RemoveVulnerability does what it says on the tin.
+//
+// Will cause copying if the vulnerability is not the most recent returned from
+// NewVulnerability.
+func (e *ecs) RemoveVulnerability(v *claircore.Vulnerability) {
```

Comment:
Does this need to be public?

Reply:
The receiver type is unexported, so this just keeps with the convention.